ShopifySharp/Filters/ListFilter.cs

```diff
@@ -40,5 +40,13 @@ public class ListFilter : CountFilter
         /// </summary>
         [JsonProperty("order")]
         public string Order { get; set; }
+
+        /// <summary>
+        /// An optional field name to include fulfillment services from entire store.
+        /// Any valid values are null and "all"
+        /// </summary>
+        [JsonProperty("scope")]
+        public string Scope { get; set; }
```

Comment:
Since the `ListFilter` class is used and inherited in several different places, could you create a new class named `FulfillmentServiceListFilter` that inherits from this `ListFilter` and adds the `Scope` property? That way we don't have the scope property showing up for unrelated services that also use this `ListFilter`.

It would look something like this:

```cs
public class FulfillmentServicesFilter : ListFilter 
{
    /// summary here
    [JsonProperty("scope")]
    public string Scope { get; set; }
}
```

Reply:
Yes, only scope and none of the other parameters. I'll make it a parameter for the method then.